Std/Data/Nat/Basic.lean

```diff
@@ -126,3 +126,12 @@ where
     else
       guess
 termination_by iter guess => guess
+
+/-!
+### testBit
+We define an operation for testing individual bits in the binary representation
+of  a number.
+-/
+
+/-- `testBit m n` returns whether the `(n+1)ˢᵗ` least significant bit is `1` or `0`-/
```

Comment:
```suggestion
/-- `testBit m n` returns whether the `(n+1)`ˢᵗ least significant bit is `1` or `0`-/
```
(It's not code, so outside the code block? I'm also dubious the superscript helps readability.